data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1625 @@
+			`	He gestures take on a confident tone. "Which makes sense, since our life experiences shape our perceptions. Still, as to what it was..." He pauses, gathering his thoughts. "Our best guess is that this was first contact with some sort of entity." He grins at that. "I know, stating the obvious. But anyway, Dusk thinks that it might be some sort of solitary entity, as it did not use words or meanings, despite having access to our minds. As such, it uses internals to convey meaning instead of words." He glances at a map of the system. "There is, however, no indication as to where it came from.`
+			choice
+				`	"How do we get back?"`
+					goto cutpower
+				`	"What happened to the long range communication?"`
+			`	"Well, that is a good question. We have, and still are, running diagnostics on them, and so far they are passing all their checks. The system safeties, isolation mechanisms, security monitoring, tamper-proofing, self-destruct systems, backup hardening..." He shakes his head, stymied.`
+			`	"Everything looks in perfect working order. The only problem is that the quantum pairs at the heart just do not seem to have a pair anymore. In terms of quantum mechanics, it seems that the rest of the universe quite simply does not exist."`
+			choice
+				`	"That seems problematic. How do we get back?"`
+					goto cutpower
+				`	"Did we destroy the universe by powering up this station?"`
+			`	Gavrill smiles with amusement. "It is certainly a possibility. However, the fact that it was, as best we can tell, powered up previously without destroying the universe as far as we can tell suggests that it transported us rather than destroy everything else."`
+			`	He frowns at the communication panel. "Although everything I know about quantum mechanics refuses to countenance something being able to leave the universe so absolutely like this." He smiles again. "But that is the way of science! Finding new things in the universe that tells us that what we know is not so absolute."`
+			choice
+				`	"I think we should focus on getting back."`
+					goto cutpower
+				`	"Should we be worried?"`
+			`	Gavriil gestures neutrally "Well, I am worried, of course. I would be devastated to find out I had destroyed humanity, and everything else in existence." He looks somber, but then perks up again. "But if everything has been annihilated, there is nothing I can do about it. So I shall focus on learning what I can, doing what must be done, and enjoy the fact that the universe has inexplicably revealed once again that the sum of all our knowledge is merely the introductory metaphor to help us grasp how things work." His gestures abruptly change tone. "Speaking of which, we need to report what we have learned to the rest, which means leaving this place."`
```

Comment:
```suggestion
			`	Gavriil gestures neutrally. "Well, I am worried, of course. I would be devastated to find out I had destroyed humanity, and everything else in existence." He looks somber, but then perks up again. "But if everything has been annihilated, there is nothing I can do about it. So I shall focus on learning what I can, doing what must be done, and enjoy the fact that the universe has inexplicably revealed once again that the sum of all our knowledge is merely the introductory metaphor to help us grasp how things work." His gestures abruptly change tone. "Speaking of which, we need to report what we have learned to the rest, which means leaving this place."`
```
`enjoy the fact that the universe has inexplicably revealed once again that the sum of all our knowledge is merely the introductory metaphor to help us grasp how things work.`
Really like that line. Nicely done!